volk.c

```diff
@@ -79,7 +78,7 @@ void volkLoadDeviceTable(struct VolkDeviceTable* table, VkDevice device)
 	volkGenLoadDeviceTable(table, device, vkGetDeviceProcAddrStub);
 }
 
-static void volkGenLoadLoader(void* context, PFN_vkVoidFunction (*load)(void*, const char*))
```

Comment:
Does the vkGetInstanceProcAddr global have to set here as well? I noticed volkInitialize does that as well.

Reply:
I think it would be more consistent to set it as well - instead of exposing volkGenLoadLoader externally I'd expose a new name like volkInitializeCustom, that internally calls volkGenLoadLoader, and probably manually initializes vkGetInstanceProcAddr using the same callback.